### PR TITLE
Eslint website

### PR DESCRIPTION
**Pull request by sedubois**
Pushing the stuff done so far (e.g doesn't have no-commonjs rule)

### CODE REVIEW

File: frontend/src/reducers/form.js [hunk 1]
```diff
@@ -42,29 +41,34 @@ const profileInitialState = {
 
 function profile(state=profileInitialState, action={}) {
   switch (action.type) {
-    case constants.SET_EDIT_MODE_PROFILE:
-      if (!action.isEditMode) {
-        return merge({}, profileInitialState, { isEditMode: action.isEditMode });
-      }
-      return merge({}, state, { isEditMode: action.isEditMode });
+    // TODO constants.SET_EDIT_MODE_PROFILE doesn't exist, should it?
```
Comment (asood123) [inline frontend/src/reducers/form.js]: you can remove all of the constants from this reducer that don't exist. `APPEND_PROFILE_FORM` is the only one we are using.

File: frontend/src/reducers/session.js [hunk 2]
```diff
@@ -21,8 +21,9 @@ export default function session(state={
         isAuthenticated: false
       });
 
-    case constants.SHOW_POPOVERMENU:
-      return merge({}, state, { hasPopOverMenuOpen: action.hasPopOverMenuOpen });
+    // TODO constants.SHOW_POPOVERMENU doesn't exist, should it?
```
Comment (asood123) [inline frontend/src/reducers/session.js]: nope. pls remove. We copied some of this code from `app` when we originally split the two. leftovers.

File: package.json [hunk 3]
```diff
@@ -12,7 +12,7 @@
     ]
   },
   "scripts": {
-    "start": "NODE_CONFIG_DIR=./server/config node server",
+    "start": "NODE_CONFIG_DIR=./server/config babel-node server",
```
Comment (asood123) [inline package.json]: I thought we aren't supposed to use `babel-node` in production?

File: package.json [hunk 4]
```diff
@@ -58,6 +58,9 @@
     "cssnano": "3.5.2",
     "ejs": "2.4.1",
     "envify": "3.4.0",
+    "eslint-plugin-import": "^1.14.0",
```
Comment (asood123) [inline package.json]: can you freeze all of these?

File: package.json (hunk 3 above)
Comment (sedubois) [inline package.json]: Will fix this in next PR https://github.com/OpenCollective/opencollective-website/pull/427

**Pull request merged**